src/routes.php

```diff
@@ -8,4 +10,14 @@
     Route::get('/pages/{slug}/edit', '\Kirschbaum\LaravelSparkPages\PageController@edit');
 });
 
-Route::get('/{slug1}/{slug2?}', '\Kirschbaum\LaravelSparkPages\PageController@show');
+Route::group(['middleware' => ['web']], function () {

```

Comment:
Definitely an interesting approach here. I might try to get more of this logic into a controller.